### PR TITLE
update ks version; run ks apply following config order

### PR DESCRIPTION
**Pull request by kunmingg**
1. update ks version in bootstrap to 0.11.0
2. run ks apply for each component following orders in config file
fix https://github.com/kubeflow/kubeflow/issues/1006.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1008)

### CODE REVIEW

File: bootstrap/cmd/bootstrap/app/server.go [hunk 1]
```diff
@@ -534,19 +533,23 @@ func Run(opt *options.ServerOption) error {
...
+			rawCmd := fmt.Sprintf(
+				"ks apply default -c %v --token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)",
+				component.Name)
+			applyCmd := exec.Command("bash", "-c", rawCmd)
+
+			var out bytes.Buffer
+			var stderr bytes.Buffer
+			applyCmd.Stdout = &out
+			applyCmd.Stderr = &stderr
+			if err := applyCmd.Run(); err != nil {
```
Comment (ankushagarwal) [inline bootstrap/cmd/bootstrap/app/server.go]: Can you print both stdout and stderr regardless of whether there was an error or not?

It helps understand what's going right/wrong

Comment (kunmingg) [reply bootstrap/cmd/bootstrap/app/server.go]: updated

**Pull request merged**